src/components/search/data/hooks.js

```diff
@@ -34,11 +54,20 @@ export const useDefaultSearchFilters = ({
       if (subscriptionPlan && subscriptionLicense?.status === LICENSE_STATUS.ACTIVATED) {
         catalogs.push(subscriptionPlan.enterpriseCatalogUuid);
       }
+
+      if (
+        features.FEATURE_BROWSE_AND_REQUEST
+        && subsidyRequestConfiguration?.subsidyRequestsEnabled
+        && catalogsForSubsidyRequests.length > 0
+      ) {
+        catalogs.push(...catalogsForSubsidyRequests);
+      }
+
       if (catalogs.length > 0) {
         return getCatalogString(catalogs);
       }
 
-      // If learner has no subsidy available to them, show all enterprise catalogs
+      // If learner has no accessible catalogs, show all enterprise catalogs
```

Comment:
This is a strange sentence to read :)

Is something like "If the learner is not confined to certain catalogs via the enterprise's current configuration, the default search filter should be scoped to all of the enterprise's catalogs." more accurate?

Reply:
nice